notebooks/official/model_registry/get_started_with_model_registry.ipynb

```diff
@@ -0,0 +1,1061 @@
+{
```

Comment:
<p>Should we update the licence year?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
no, rule is the year the notebook was authored